Core/CMP API Specification.md

```diff
@@ -566,10 +564,11 @@ gppVersion : Number, // The version number parsed from the header
 
 sectionList : Array of Number, // the sections contained within the encoded GPP string as parsed from the header
 
-applicableSection: Array of Number, // Section ID considered to be in force for this transaction. In most cases, this field should have a single section ID. In rare occasions where such a single section ID can not be determined, the field may contain up to 2 values.
+applicableSection: Array of Number, // Section ID considered to be in force for this transaction. The value should be 0 during stub/load. In most cases, this field should have a single section ID. In rare occasions where such a single section ID can not be determined, the field may contain up to 2 values. When no section is applicable, the value will be -1.
```

Comment:
applicableSection**s**

Reply:
I am not sure it makes sense to say it is rare to have two sections; it is very easy to imagine three sections usually applying for a majority of an EU publisher (who has decided gdpr always applies) who publishes both section 6 and section 7 for their majority usa audience.